flowsa/__init__.py

```diff
@@ -7,73 +7,81 @@
 """
 
 import pandas as pd
-from flowsa.common import fbaoutputpath, fbsoutputpath, datapath, log
+import os
+from flowsa.common import outputpath, fbaoutputpath, fbsoutputpath, datapath, log
 from flowsa.flowbyfunctions import collapse_fbs_sectors, filter_by_geoscale
 from flowsa.datachecks import check_for_nonetypes_in_sector_col, check_for_negative_flowamounts
 
-
-def getFlowByActivity(flowclass, years, datasource, geographic_level='all'):
+def getFlowByActivity(flowclass, years, datasource, geographic_level='all', file_location='local'):
     """
     Retrieves stored data in the FlowByActivity format
     :param flowclass: list, a list of`Class' of the flow. required. E.g. ['Water'] or
      ['Land', 'Other']
-    :param year: list, a list of years [2015], or [2010,2011,2012]
+    :param year: list, a list of years [2015], or [2010,20flowclass, years, datasource, geographic_level='all', file_location='local'11,2012]
```

Comment:
typo?

Reply:
Ha! Thank you, corrected this typo (and the next)